src/main/java/org/candlepin/pki/certs/AnonymousCertificateGenerator.java

```diff
@@ -317,16 +313,15 @@ private Set<X509Extension> prepareV3ByteExtensions(org.candlepin.model.dto.Produ
     }
 
     private byte[] createContentAccessDataPayload(Consumer consumer,
-        Map<org.candlepin.model.Content, Boolean> ownerContent, PromotedContent promotedContent) {
+        Map<String, ProductContent> activeContent, PromotedContent promotedContent) {
 
         String consumerUuid = consumer != null ? consumer.getUuid() : null;
         log.info("Generating SCA payload for consumer \"{}\"...", consumerUuid);
```

Comment:
Similarly, lets change `SCA` to `ACOA` (anonymous cloud offering access) here? the current log string is leftover from when this code was shared between anonymous and SCA certs, and leaving it like this is innacurate and would confuse during debugging.

Reply:
Then I suppose this should also be `ACCA` (Anonymous Cloud Content Access`) instead of `SCA`